sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -217,20 +208,29 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
     keyPhrases: string[];
 }
 
-// @public (undocumented)
-export interface HealthcareAssertion {
-    association?: Association;
-    certainty?: Certainty;
-    conditionality?: Conditionality;
-}
-
 // @public
 export interface HealthcareEntity extends Entity {
-    assertion?: HealthcareAssertion;
+    assertion?: HealthcareEntityAssertion;
     dataSources: EntityDataSource[];
     normalizedText?: string;
 }
 
+// @public (undocumented)
+export interface HealthcareEntityAssertion {
+    association?: HealthcareEntityAssociation;
+    certainty?: HealthcareEntityCertainty;
+    conditionality?: HealthcareEntityConditionality;
+}
+
+// @public
+export type HealthcareEntityAssociation = "subject" | "other";
```

Comment:
Does JS/TS need to capitalize the first letter, such as "Subject", "Other"?

Reply:
I am just following the swagger, does this needs to be fixed in the swagger?